appyters/PrismEXP/appyter.json

```diff
@@ -14,7 +14,9 @@
     "url": "https://github.com/MaayanLab/appyter-catalog",
     "tags": [
       "PrismEXP",
-      "gene function prediction"
+      "gene function prediction",
```

Comment:
Consider adopting tags that already exist on appyters https://appyters.maayanlab.cloud